examples/nextjs-keystone/src/keystone/schema.ts

```diff
@@ -19,7 +19,7 @@ const permissions = {
 export const lists: Lists = {
   User: list({
     // readonly for demo purpose
-    access: permissions.readOnly,
+    access: allowAll,
```

Comment:
We probably have to change the permission to `readOnly`. This example is deployable to vercel in one click using the button in the docs by anyone. `allowAll` gives CRUD access to all fields and it's probably not safe for an example deployed on vercel with public write access.

Reply:
Done